Library/Homebrew/cmd/install.rb

```diff
@@ -251,8 +258,10 @@ def install
             #{msg}, it's just not linked.
             You can use `brew link #{old_formula.full_name}` to link this version.
           EOS
+        elsif args.quiet?
```

Comment:
Should this be only in the `elsif` case?

Reply:
Yup, we want `msg` to be set to the default otherwise  👍🏻